src/mscorlib/src/System/RuntimeHandles.cs

```diff
@@ -912,7 +912,7 @@ internal static Utf8String GetUtf8Name(RuntimeMethodHandleInternal method)
         [DebuggerStepThroughAttribute]
         [Diagnostics.DebuggerHidden]
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
-        internal extern static object InvokeMethod(object target, object[] arguments, Signature sig, bool constructor);
+        internal extern static object InvokeMethod(object target, object[] arguments, Signature sig, bool constructor, bool doNotWrapExceptions);
```

Comment:
I think it would be good to make the new argument a positive (`wrapExceptions`) to match the CreateInstance method. It's easier to follow without the negative.

Reply:
Other than the "doNotWrapExceptions" vs. "wrapExceptions" issue, this looks good to me.